code/src/pwm.cpp

```diff
@@ -154,13 +154,17 @@ namespace R2D2::pwm_lib {
         }
     }
     
-    void pwm_c::set_polarity(polarity new_polartiy) {
-        ch_polarity = new_polartiy;
-        if (ch_polarity == polarity::NEGATIVE){
+    void pwm_c::set_polarity(polarity new_polarity) {
```

Comment:
Not really meant for this PR? But alright

Reply:
My bad I made the branch from master..